u_handshake_client.go

```diff
@@ -97,7 +97,7 @@ func (hs *clientHandshakeStateTLS13) decompressCert(m utlsCompressedCertificateM
 	rawMsg[3] = uint8(m.uncompressedLength)
 
 	n, err := decompressed.Read(rawMsg[4:])
-	if err != nil {
+	if err != nil && !errors.Is(err, io.EOF) {
```

Comment:
As noted in the commit message, we check that `n` equals the expected value below, so we can safely ignore EOF here.

Reply:
Great! Thanks for the clarification!